cypress-custom/integration/fee.test.ts

```diff
@@ -1,62 +1,71 @@
 import { ChainId, WETH } from '@uniswap/sdk'
 
+const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
+const RINKEBY = ChainId.RINKEBY.toString()
 const FEE_QUERY = `https://protocol-rinkeby.dev.gnosisdev.com/api/v1/tokens/${WETH[4].address}/fee`
 const FEE_QUOTES_LOCAL_STORAGE_KEY = 'redux_localstorage_simple_fee'
 
+function _assertFeeData(fee: any): void {
+  expect(fee).to.have.property('minimalFee')
+  expect(fee).to.have.property('feeRatio')
+  expect(fee).to.have.property('expirationDate')
+}
+
+function _getLocalStorage(): Cypress.Chainable<Storage> {
+  return cy.window().then(window => window.localStorage)
```

Comment:
why not use the alias as i mentioned in #121

Reply:
Replied there